modules/custom/openy_mappings/src/Entity/Mapping.php

```diff
@@ -149,7 +149,7 @@ public function isPublished() {
    * {@inheritdoc}
    */
   public function setPublished($published) {
-    $this->set('status', $published ? NODE_PUBLISHED : NODE_NOT_PUBLISHED);
+    $this->set('status', $published ? \Drupal\node\NodeInterface::PUBLISHED : \Drupal\node\NodeInterface::NOT_PUBLISHED);
```

Comment:
Namespaced classes/interfaces/traits should be referenced with use statements

Reply:
fixed https://github.com/ymcatwincities/openy/pull/2174/commits/839562bfa412cf71c9862ca9d9ce54b72f92ff06